Sources/ArcGISToolkit/Components/Popups/PopupMedia/Images/AsyncImageViewModel.swift

```diff
@@ -107,8 +107,8 @@ struct LoadImageError: Error {
 
 extension LoadImageError: LocalizedError {
     public var errorDescription: String? {
-        return NSLocalizedString(
-            "The URL could not be reached or did not contain image data",
+        return String(
+            localized: "The URL could not be reached or did not contain image data",
             bundle: .toolkitModule,
             comment: "No Data"
```

Comment:
I recently (re)watched [Build global apps: Localization by example](https://developer.apple.com/wwdc22/10110). In it, the presenter explained that a good localized string comment has three components:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/2257493/76aaa870-5fde-445d-8d10-c56baa85ce37)

I would suggest expanding this comment to include both what the string is used for and the context. Perhaps something like "Description of error thrown when a remote image could not be loaded."